src/types.ts

```diff
@@ -3192,12 +3193,21 @@ export enum HostEvent {
      * Trigger the **Download PDF** action on an embedded Liveboard,
      * visualization or Answer.
      *
+     * @param - `vizId` refers to the answer id in spotter Embed, it is required in spotter Embed.
```

Comment:
`vizId` refers to the Answer ID in Spotter Embed and is required in Spotter Embed.